src/hotspot/share/utilities/vmError.cpp

```diff
@@ -106,6 +106,9 @@ static const char* env_list[] = {
   "JAVA_HOME", "JAVA_TOOL_OPTIONS", "_JAVA_OPTIONS", "CLASSPATH",
   "PATH", "USERNAME",
 
+  // UNIX platforms, fontconfig related
+  "XDG_CACHE_HOME", "XDG_CONFIG_HOME", "FC_LANG",
```

Comment:
This seems a very limited selection - why only these three? (And I find it hard to imagine how these particular ones might lead to crashes - FONTCONFIG_USE_MMAP seems much more interesting in that regard).